pkg/plugins/policies/meshtrafficpermission/xds/v3/rbac_configurer_test.go

```diff
@@ -0,0 +1,256 @@
+package v3_test
```

Comment:
maybe get rid of v3 we've deprecated v2 and Matt Klein himself said that there'll likely never be a v4

Reply:
ok, I don't mind having just `meshtrafficpermission/xds`